guide/source/program_administrator/introduction.rst

```diff
@@ -87,7 +89,7 @@ will want to familiarize yourself with this group, as they can serve as an
 invaluable resource to assist in the implementation of the BPI data standards,
 facilitate changes in the standard, and answer questions that you may have
 about the standards. To contact the BPI WG-5 or to get involved, email them at
-Julie.Caracino@nyserda.ny.gov.
+Julie.Caracino@nyserda.ny.gov. [IT WOULD BE PREFERABLE TO INCLUDE A GENERIC EMAIL ADDRESS LIKE HPXML@BPI.ORG INSTEAD OF A PERSON'S EMAIL ADDRESS IN A GUIDE LIKE THIS. OTHERWISE, WE COULD HAVE THEM GO TO HOMEPERFORMANCE@ENERGYSTAR.GOV]
```

Comment:
@nmerket did you discuss with Robin who would may that email?   Would that be HPC staff?